app/controllers/rx_controller.rb

```diff
@@ -4,13 +4,25 @@
 class RxController < ApplicationController
   include ActionController::Serialization
 
-  skip_before_action :authenticate
+  before_action :authorize_rx
   before_action :authenticate_client
 
   protected
 
   def client
-    @client ||= Rx::Client.new(session: { user_id: ENV['MHV_USER_ID'] })
+    @client ||= Rx::Client.new(session: { user_id: mhv_correlation_id })
+  end
+
+  def authorize_rx
+    mhv_correlation_id || raise_access_denied
```

Comment:
I think it would be bad for front end to loop indefinitely, they should have some reliable expectation of the loa level, and work off of that. I'm leaning towards simplicity here, but if we need a more descriptive error message we can address later.